src/contracts/StablecoinConverter.d.ts

```diff
@@ -0,0 +1,188 @@
+/* Generated by ts-generator ver. 0.0.8 */
+/* tslint:disable */
+
+import BN from 'bn.js'
+import { Contract, ContractOptions } from 'web3-eth-contract'
+import { EventLog, Callback, EventEmitter } from 'web3-core'
+import { TransactionObject, BlockType } from 'web3/eth/types'
```

Comment:
These are old from `@types/web3` and they are slightly wrong. Now that types are bundled with the actual web3 everything is different and you can't just take typechain results at face value.
You can have a look at how I did that in [dex-react](https://github.com/gnosis/dex-react/blob/real_erc20_api/src/types/index.ts#L175)

Reply:
Yes, this part I was planning to shamelessly take your type definition from dex-react while we don't make it a shared module

Still pending this part

Reply:
This will be addressed in an upcomming PR, but the final solution should come with this https://github.com/gnosis/dex-react/issues/200